alf/trainers/policy_trainer.py

```diff
@@ -465,7 +465,7 @@ def _eval(self):
             self._eval_env.batch_size)
         episodes = 0
         while episodes < self._num_eval_episodes:
-            time_step, policy_state, trans_state, _ = _step(
+            time_step, policy_state, trans_state, _, _ = _step(
```

Comment:
Can just let _step() return policy_step to reduce number of return arguments

Reply:
done